docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -1,83 +1,100 @@
+The Alpha release of the Cassandra Source Connector (CSC) is a new, open-source product from DataStax.
 
-=== What are the system requirements for Cassandra Data Connector?
+With CDC, your apps can consume data from an Apache Cassandra table and submit the updates onto a Pulsar topic.  This topic can then be consumed by external targets such as Elastic, Snowflake, and other platforms. By design, the CSC is meant to be a simple connector with a 1:1 correspondence between a Cassandra table and a single Pulsar topic.
 
-Cassandra Source Connector requires additional resources depending on your write throughput.
-For each cassandra write, each Cassandra replica send one message to your streaming platform (Pulsar or Kafka), and
-execute at least one QUORUM read on your
+The CSC is composed of two parts:
 
-=== What components and features are provided by Cassandra Data Connector?
+* The Cassandra Source Connector, which transmits Change Data Capture (CDC) updates from the Cassandra table to the streaming service.
+* The Pulsar CSC Connector, which verifies the CSC events and stores the final data changes into a Data topic.
 
-Cassandra Data Connector provides:
+=== What are the system requirements for the CSC?
+
+The CSC requires additional resources depending on your write throughput.
+
+For each Cassandra write, each Cassandra replica sends one message to your Pulsar service, and executes at least one QUORUM read on your Cassandra cluster.
+
+=== What components and features are provided by Cassandra Source Connector?
+
+Cassandra Source Connector provides:
 
 * Cassandra, Elastic, Snowflake, Kinesis, Kafka, and JDBC connectors
 * A management dashboard
 * A monitoring and alerting system
 
-=== Which versions of Apache Pulsar does DataStax Cassandra Data Connector support?
+=== Which versions of Apache Pulsar does DataStax Cassandra Source Connector support?
 
-Cassandra Data Connector supports Apache Pulsar 2.7.0 and later.
+Cassandra Source Connector supports Apache Pulsar 2.7.0 and later.
 
-=== What does Cassandra Data Connector provide that I cannot get with open-source Apache Pulsar?
+=== What does the CSC provide that is not provided by Apache Pulsar?
 
-DataStax Cassandra Data Connector implements, in effect, the reverse of Apache Pulsar or DataStax CassaNdra data Connector. With CSC, the Source takes data from Cassandra and puts that date onto onto a Pulsar topic. From there, the data can be published to external platforms like Snowflake, Elastic, and so on.
+DataStax Cassandra Source Connector implements the reverse of the DataStax Cassandra Source Connector, which stores Pulsar topic updates into Cassandra.  The CSC takes data from a Cassandra table and stores that data into a Pulsar topic.  From there, the data can be published to external platforms like Snowflake, Elastic, and so on.
```

Comment:
The first sentence doesn't make sense here. Should the second reference to CSC actually be `Pulsar Source Connector`?

Reply:
I got it figured out - the DataStax CSC is the reverse of the Pulsar Connector.  AKA the CSC is the reverse of the APC: https://downloads.datastax.com/#apc .

I've updated this to correct the issue, and provided a link to the APC so it makes more sense.